perf/istio/external-traffic/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I would like to avoid creating additional installers here - in the new 'advanced installer' we'll have support for multiple control plane namespaces with different options. 

Do we have any per-pod annotation to allow this to be configured selectively ?

Reply:
Is this referring to having to set `global.outboundTrafficPolicy.mode=ALLOW_ANY`?

It looks like this is only configured at the mesh level, not per-pod.

https://github.com/istio/istio/blob/14777199b85ba56ebca2a4516afff33c97199eb4/pilot/pkg/networking/core/v1alpha3/listener.go#L199-L205

Not sure if this was meant to be commented on the `setup.sh` and/or if there was confusion, but the `setup.sh` just `kubectl apply`s the required files to get  the scenario setup. It could probably be a little more clear. The intention is to have a top level `/perf/istio/setup.sh` that will set up and install istio, then you can run what scenarios you want (`http10/setup.sh && external-traffic/setup.sh`, etc).